src/app/core/services/filter-search.service.ts

```diff
@@ -21,6 +21,6 @@ export class FilterSearchService {
    * @param value
    */
   transformForSearch(value: string) {
-    return value.toLowerCase().replace(/\s/g, '').trim();
+    return value.toLowerCase().replace(/\s+/g, '');
```

Comment:
Is it really required to remove all spaces? Or only at the start and at the end of string?